custom_components/coway/sensor.py

```diff
@@ -352,7 +356,94 @@ def available(self) -> bool:
         """Return true if purifier is connected to Coway servers."""
 
         if self.purifier_data.network_status:
-            return True
+            if self.purifier_data.particulate_matter_10.isnumeric():
+                return True
+            else:
+                return False
+        else:
+            return False
```

Comment:
See comment above - this sensor won't apply to the iconS in the future so the numeric check isn't necessary since other purifiers always return a value.